src/webview_provider/altimateWebviewProvider.ts

```
@@ -410,7 +414,7 @@
           // If config is for preview feature, then check keys
           const shouldUpdate =
             !params.isPreviewFeature ||
-            this.altimateRequest.handlePreviewFeatures();
+            this.altimateAuthService.handlePreviewFeatures();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)